ci/praktika/json.html

```diff
@@ -715,20 +715,28 @@
             const thead = document.createElement('thead');
             const tbody = document.createElement('tbody');
 
+            let existing_columns = {};
+            columns.forEach(column => {
+                existing_columns[column] = results.some(result => result[column]);
+            });
+
             // Create table headers based on the fixed columns
             const headerRow = document.createElement('tr');
             columns.forEach(column => {
+                if (!existing_columns[column]) {
+                    return;
+                }
```

Comment:
```suggestion
            existing_columns.forEach(column => {
```

Reply:
Do you suggest removing check 

```cpp
if (!existing_columns[column]) {
    return;
}
```

?
Or is it diff messed up?

Reply:
Yes, if I get the code right, you could just iterate column names directly from the `existing_columns` array. Anyway it is minor. Ignore it if think it is better to keep things as they are now.